src/SqlAsyncCollector.cs

```diff
@@ -445,8 +475,9 @@ AS NewData
             /// <param name="sqlConnection">An open connection with which to query SQL against</param>
             /// <param name="fullName">Full name of table, including schema (if exists).</param>
             /// <param name="logger">ILogger used to log any errors or warnings.</param>
+            /// <param name="columnNames">Column names of the table</param>
```

Comment:
This isn't accurate though, right? It's actually column names from the object (that should map to ones from the table, but that's not really important here)